tests/dummy/public/test-forms/form-builder/form.ttl

```diff
@@ -0,0 +1,370 @@
+
+ext:formNodesTypeF a form:Field ;
+  sh:name "Field type" ;
+  sh:order 3 ;
+  sh:path form:displayType ;
+  form:help """
+    e.g. text input, date, dropdown...
+   """;
+  form:displayType displayTypes:conceptSchemeSelector ;
+  form:options """{"conceptScheme":"http://lblod.data.gift/concept-schemes/c5a91bd7-3eb5-4d69-a51b-9bac6bf345f6","searchEnabled":true}""" ;
+  sh:group ext:formFieldPg .
+
+ext:formNodeHelpTextF a form:Field;
+  sh:name "Helptext" ;
+  sh:order 4 ;
+  form:help """
+    Order of the section, calculated from top to bottom.
+   """;
+  sh:path form:help ;
+  form:displayType displayTypes:textArea;
+  sh:group ext:formFieldPg .
+
+ext:formNodeOptionsF a form:Field;
+  sh:name "Extra configuration options as JSON" ;
+  sh:order 5 ;
+  sh:path form:options ;
+  form:displayType displayTypes:textArea;
+  sh:group ext:formFieldPg .
+
+ext:formNodePropertyGroupF a form:Field;
+  sh:name "Link to section" ;
+  form:help """
+   This is required. If you dont specify a section, the field will not be visible.
+   """;
+  sh:order 6 ;
+  sh:path sh:group ;
+  sh:group ext:formFieldPg .
+
+##########################################################
+#  Generator: formNodesForm
+##########################################################
+ext:formNodesGenerator a form:Generator;
+  form:prototype [
+    form:shape [
+      a form:Field ;
+      sh:name "Field name";
+      sh:path [];
+      form:displayType displayTypes:defaultInput
+    ]
+  ].
+
+##########################################################
+#  listing-scope:
+##########################################################
+ext:formListingS a form:Scope;
+  sh:path form:includes;
+  form:constraint [
+     a sh:NodeShape ;
+     sh:property [
+       sh:path rdf:type ;
+       sh:targetNode form:Listing
+     ];
+  ].
+
+##########################################################
+#  listing: formListingL
+##########################################################
+ext:formListingL a form:Listing;
+  form:each ext:formListingFormItem;
+  form:scope ext:formListingS;
+  form:createGenerator ext:formListingGenerator;
+  form:canAdd true;
+  form:addLabel "Add new Listing";
+  form:canRemove true;
+  form:canChangeOrder true;
+  sh:group ext:mainPg;
+  sh:order 3 .
+
+##########################################################
+#  listing: formListingL (as table)
+##########################################################
+ext:formListingTableL a form:Listing;
+  form:each ext:formTableListingFormItem;
+  form:scope ext:formListingS;
+  form:createGenerator ext:formListingTableGenerator;
+  form:canAdd true;
+  form:addLabel "Add new table listing";
+  form:canRemove true;
+  form:canChangeOrder true;
+  sh:group ext:mainPg;
+  sh:order 4 .
+
+##########################################################
+#  Subform: ext:formListingFormItem
+##########################################################
+ext:formListingFormItem a form:SubForm;
+  sh:name "Listing";
```

Comment:
this is a change from how the actual form builder organizes its forms. By putting the name at the form level, the controls are nicely wrapped around the title of the form. Otherwise the title (of the propertygroup as configured in the real form builder) appears below the controls and looks out of place.